packs/data/extinction-curse-bestiary.db/bitter-truth-bandit.json

```
@@ -864,7 +866,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>A bandit deals an additional 2d6 precision damage to flat-footed creatures.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
+                    "value": "<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
```

```suggestion
                    "value": "<p>A bandit deals an additional 2d6 precision damage to flat-footed creatures.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
```
Make sure you are leaving the actual book text there after you refresh an item.